src/components/backend-ai-storage-list.ts

```diff
@@ -2508,7 +2524,16 @@ export default class BackendAiStorageList extends BackendAIPage {
     for (let i = 0; i < this.uploadFiles.length; i++) {
       this.fileUpload(this.uploadFiles[i]);
     }
-    (this.shadowRoot?.querySelector('#fileInput') as HTMLInputElement).value = '';
+    if (isEmptyFileIncluded) {
+      this.notification.text = _text('data.explorer.EmptyFilesAreNotUploaded');
+      this.notification.show();
+    }
+    if (isFolderUpload) {
+      this.notification.text = _text('data.explorer.EmptyFoldersAreNotUploaded');
```

Comment:
I found that It skips uploading not only for the empty folder(s) but also for the empty file(s). IMO, It's better to inform users about both conditions at once.